ansible/group_vars/raspbian

```diff
@@ -1,10 +1,17 @@
 ---
+
+os_target: raspbian
+
 ansible_user: pi
 
+# Since the raspbian image may be used in more than one device, let nginx set the number of processes
+nginx_worker_processes: auto
```

Comment:
Couldn't we move `nginx_worker_processes: auto` into `group_vars/all` given it’s the same everywhere?